pyprep/utils.py

```diff
@@ -20,6 +21,71 @@ def _intersect(list1, list2):
     return list(set(list1).intersection(set(list2)))
 
 
+def _mat_quantile(arr, q, axis=None):
+    """Calculate the numeric value at quantile (q) for a given distribution.
```

Comment:
Just a very nitpicky and unimportant note on numpydoc style: when you describe parameters in the docstring, you should wrap them in single backticks, like:

```suggestion
    """Calculate the numeric value at quantile (`q`) for a given distribution.
```

See the "4. parameters" section here: https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
Good catch, will fix.